entity-types/infra-kubernetes_pod/definition.yml

```diff
@@ -62,3 +62,33 @@ synthesis:
       created_by_kind:
         # from kube-state-metrics kube_pod_info
         entityTagNames: [k8s.createdKind]
+
+  - compositeIdentifier:
+      separator: ":"
+      attributes:
+        - k8s.cluster.name
+        - k8s.namespace.name
+        - k8s.pod.name
+    encodeIdentifierInGUID: true
+    name: pod
+    conditions:
+      # identifier attributes
+      - attribute: k8s.pod.name
+        present: true
+      - attribute: k8s.namespace.name
+        present: true
+      - attribute: k8s.cluster.name
+        present: true
+      # open telemetry
+      - attribute: newrelic.source
+        value: 'api.traces.otlp'
+      # ebpf data
```

Comment:
Do we need this attribute? i.e. there might be a legitimate case where we want to synthesize this entity from metrics vs. traces.

Reply:
Agreed. Removed.